lib/LedgerSMB/Template/XLSX.pm

```diff
@@ -65,7 +65,8 @@ my $rowcount;
 my $currcol;
 
 sub _worksheet_handler {
-    $rowcount = -1;
+    $rowcount = 0;
+    -- $rowcount;
```

Comment:
@ylavoie I think the code can be completely removed; the rowcount isn't used anywhere, is it?